src/main/java/homework_2/random_chars_table/RandomCharsTable.java

```diff
@@ -0,0 +1,79 @@
+package homework_2.random_chars_table;
+
+import java.util.*;
+
+public class RandomCharsTable {
+
+    protected static final String ANSI_RED = "\u001B[31m";
+    protected static final String ANSI_RESET = "\033[0m";
+
+    public void start() {
+        Scanner systemScanner = new Scanner(System.in);
+        Scanner stringScanner;
+        char[][] chars;
+        String parity;
+        List<Character> characterList;
+
+        System.out.println("Enter width, height and parity");
+        String s = systemScanner.nextLine();
+        if (s.matches("^(([1-9][0-9]*\\s){2})(odd|even)")) {
+            stringScanner = new Scanner(s);
+            chars = RandomCharsTableCreator.initTable(stringScanner.nextInt(), stringScanner.nextInt());
+            parity = stringScanner.next();
+            characterList = checkChars(chars, parity);
+
+            printTable(chars);
+            System.out.println(parity + " letter(s) - " + getResult(characterList));
+            stringScanner.close();
+        } else {
+            System.out.println(ANSI_RED + "Passed parameters should match the format [positive integer] [positive integer] [even|odd]" + ANSI_RESET);
```

Comment:
Opt: can be class field

Reply:
for example, ERROR_MESSAGE? ("Passed parameters should ma...")